### PR TITLE
Rad patch

### PR DESCRIPTION
**Pull request by MadmanMartian**
Idea and sprite courtesy of @Blithering 

A small patch you can attach to your jumpsuit that will click and break should it absorb a certain amount of radiation, making it go from white to black.

:cl:
* rscadd: Adds Rad patches. A jumpsuit accessory that changes color when it absorbs radiation

### CODE REVIEW

**Comment by WhiteHusky** [comment]
Could be included with the closets containing radsuits?

**Comment by SonixApache** [comment]
pictures

File: code/modules/clothing/accessories/accessory.dm [hunk 1]
```diff
@@ -79,7 +79,7 @@
 	if(accessories.len)
 		. = list()
 		for(var/obj/item/clothing/accessory/accessory in accessories)
-			. += "[bicon(accessory)] \a [accessory]"
+			. += "[bicon(accessory)] \a [accessory]."
```
Comment (Exxion) [inline code/modules/clothing/accessories/accessory.dm]: Won't this put a period after *every* accessory?

File: code/modules/clothing/accessories/accessory.dm [hunk 2]
```diff
@@ -319,3 +319,51 @@
...
+	w_type = RECYK_WOOD
+
+/obj/item/clothing/accessory/rad_patch/proc/check_rads(list/arguments)
+	if(triggered)
+		return
+	var/mob/user = arguments["user"]
+	var/rads = arguments["rads"]
+	rad_absorbed += rads
+
+	if(!triggered && rad_absorbed > rad_threshold)
```
Comment (Exxion) [inline code/modules/clothing/accessories/accessory.dm]: The `!triggered` check is redundant. Also it should remove the event listener in here, unless you have a particular reason for it continuing to run the proc after the thing is already triggered.

**Comment by MadmanMartian** [comment]
spess facility ix 2017-12-12 190530

Pictures for those who want them.

White is not-triggered, black is triggered. The patch 'clicks' when it is triggered so you know to run.

**Pull request merged**